tap_github.py

```diff
@@ -31,16 +31,62 @@ def get_abs_path(path):
 def load_schemas():
     schemas = {}
 
-    with open(get_abs_path('tap_github/commits.json')) as file:
-        schemas['commits'] = json.load(file)
+    for filename in os.listdir(get_abs_path('tap_github')):
+        path = get_abs_path('tap_github') + '/' + filename
+        file_raw = filename.replace('.json', '')
+        schemas[file_raw] = json.load(open(path))
```

Comment:
Do you know if `json.load()` will close the file afterwards? I'm wondering if this loop should include:

```
with open(path) as f:
  schemas[file_raw] = json.load(f)
```

Reply:
you're right; I should either do a `with` or close it

https://stackoverflow.com/questions/22348859/will-passing-open-as-json-load-parameter-leave-the-file-handle-open